functions.sh

```diff
@@ -236,6 +236,14 @@ function json_escape()
   echo -n "$1" | python -c 'import json,sys; print json.dumps(sys.stdin.read())'
 }
 
+#
+# Get Gist API endpoint from top level API
+#
+function get_gist_api()
+{
+  curl --silent --header "$auth_header" https://$server/api/v3 | python -c 'import json,sys; print json.loads(sys.stdin.read())["gists_url"].split("{")[0]'
```

Comment:
For me this results in a call to `https://api.github.com/api/v3` which is 404. However, just plain `https://api.github.com/` returns a hash of urls that looks to be what you are after. Are you seeing something different? Does the latter URL work for you?

Reply:
Ah, yeah, I'll see if I can make that a bit a smarter without having to change the config to store a url and not the server